test/integration/targets/azure_rm_virtualmachine_scaleset/tasks/main.yml

```diff
@@ -50,14 +55,33 @@
         managed_disk_type: Standard_LRS
   register: results
 
-- name: Assert that VMSS ran
+- name: Assert that VMSS was created
+  assert:
+    that: results.changed
+
+- name: Retrieve scaleset facts
+  azure_rm_virtualmachine_scaleset_facts:
+    resource_group: "{{ resource_group }}"
+    name: testVMSS{{ rpfx }}
+    format: curated
+  register: output_scaleset
+
+- name: Modify scaleset (set upgradePolicy to Automatic and capacity to 3)
+  set_fact:
+    body: "{{ output_scaleset.ansible_facts.azure_vmss[0] | combine({'capacity': 1}, recursive=True) }}"
+
+- name: Update VMSS
+  azure_rm_virtualmachine_scaleset: "{{ body }}"
+  register: results
+
+- name: Assert that VMSS was updated
```

Comment:
Can you have some better tests for the output_scaleset result, such as verifying the structure of the result and that known values match what you expect them to be.

Reply:
yes, i will do this, as i remove update task, this is needed